pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -69,8 +84,54 @@ type KubeControllersConfiguration struct {
 	KibanaSecret                 *corev1.Secret
 }
 
-func KubeControllers(cfg *KubeControllersConfiguration) *kubeControllersComponent {
-	return &kubeControllersComponent{cfg: cfg}
+func NewCalicoKubeControllers(cfg *KubeControllersConfiguration) *kubeControllersComponent {
+	kubeControllerRolePolicyRules := kubeControllersRoleCommonRules(cfg, KubeController)
+	enabledControllers := []string{"node"}
+	if cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise {
+		kubeControllerRolePolicyRules = append(kubeControllerRolePolicyRules, kubeControllersRoleEnterpriseCommonRules(cfg)...)
+		kubeControllerRolePolicyRules = append(kubeControllerRolePolicyRules, calicoKubeControllersRoleSpecificRules()...)
```

Comment:
I assume you're talking about `calicoKubeControllersRoleSpecificRules` and `elasticsearchKubeControllersRoleSpecificRules` because other rules are common and shared by both constructors.

Reply:
Yes I was